model/src/main/kotlin/com/testerum/model/resources/http/response/xml_body/XmlRootJsObject.kt

```diff
@@ -0,0 +1,94 @@
+package com.testerum.model.resources.http.response.xml_body
+
+import jdk.nashorn.api.scripting.AbstractJSObject
+import org.jdom2.Attribute
+import org.jdom2.Content
+import org.jdom2.Document
+import org.jdom2.Element
+import org.jdom2.Text
+import org.jdom2.input.SAXBuilder
+import org.jdom2.xpath.XPathFactory
+
+const val XPATH_FUNCTION_NAME = "xpath"
+
+class XmlRootJsObject(private val xmlAsUtf8String: String) : AbstractJSObject() {
```

Comment:
I have an idea that may help us get rid of 2 JS objects (putting the parsing of the XML document in a lambda, in a companion object).

Let's do the refactoring of this PR together.